src/Membership/MembershipRequest.php

```diff
@@ -106,9 +105,6 @@ class MembershipRequest implements MembershipInterface
 
     public function __construct()
     {
-        $this->gender = Genders::MALE;
```

Comment:
maybe assign the default values on property?
`private $gender = Genders::MALE`?

Reply:
We don't want a default value anymore in this case.